exporter/metric/metric_test.go

```diff
@@ -39,14 +40,20 @@ var (
 )
 
 func TestDescToMetricType(t *testing.T) {
+	cloudMock := cloudmock.NewCloudMock()
```

Comment:
Where do you shut down this mock? See `defer mock.Shutdown()` in #66

Reply:
Nice catch, added a shutdown call below.